stdlib/websub/src/main/ballerina/src/websub/hub_service.bal

```diff
@@ -550,6 +559,21 @@ function distributeContent(string callback, SubscriptionDetails subscriptionDeta
     return;
 }
 
+function callOnDeliveryFailure(string callback, string topic, WebSubContent notification, http:Response|error response) {
+    function (string callback, string topic, WebSubContent content, http:Response|error response)?
+                               onDeliveryFailureFunc = onDeliveryFailureFunction;
+    if (onDeliveryFailureFunc is function (string callback, string topic, WebSubContent content, http:Response|error response)) {
```

Comment:
If we go with just functions as suggested above, we wouldn't need the type test. But generally, it is less expensive to check if a variable is `()` rather than a specific type, so if we are going with this approach let's invert the check.

Reply:
Moved to the approach suggested in https://github.com/ballerina-platform/ballerina-lang/pull/27571#discussion_r545610193

Reply:
Inverted the check.